bootstrap/src/contexts/ProjectList.js

```diff
@@ -20,15 +21,15 @@ const getFirstProject = projects => {
 };
 
 export const ProjectListProvider = ({ children }) => {
-    const authState = useAuth();
+    const { authStatus } = useAuth();
     const [projectListState, setProjectListState] = useState({ ...defaultSourceState });
     const [firstProjectId, setFirstProjectId] = useState(null);
 
     useEffect(() => {
-        const getProjects = async userId => {
+        const getProjects = async userInfo => {
             setProjectListState({ isLoading: true });
             try {
-                const currentProjects = await sdk.project.getProjects(userId);
+                const currentProjects = await sdk.project.getProjects((await userInfo).loginMD5);
```

Comment:
Why the `(await userInfo)` await?

Reply:
because I was passing an async function into the parameter. I can change it to `getProjects(await sdk.user.getAccountInfo());` instead.

Reply:
Also, there is an endpoint in backend SPI for the projects: backend.workspaces(), please us that here.